packages/flutter/test/foundation/diagnostics_test.dart

```diff
@@ -1914,9 +1914,11 @@ void main() {
             '--- example property at max length --',
             style: propertyStyle,
           ),
-          DiagnosticsProperty<void>(null,
-              'Message that is not allowed to wrap even though it is very long. Message that is not allowed to wrap even though it is very long. Message that is not allowed to wrap even though it is very long. Message that is not allowed to wrap.',
-              allowWrap: false),
```

Comment:
Regarding the failing test the fix should be to replace `<void>` with `<String>`